### PR TITLE
Feature/datastacks testing

### PR DESCRIPTION
**Pull request by satenderrathee**
datastacks pipeline

### CODE REVIEW

**Review by motylp — APPROVED** [review]

File: datastacks/datastacks-pipeline.yml [hunk 1]
```diff
@@ -0,0 +1,62 @@
+name: "$(Build.SourceBranchName)-init"
+
+#trigger:
```
Comment (motylp) [inline datastacks/datastacks-pipeline.yml:3]: remove comment?

File: datastacks/datastacks-pipeline.yml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+              poetry install 
+            displayName: Install Dependencies
+            workingDirectory: $(self_repo_dir)
+          - bash: |
+              poetry run pytest -rf -v datastacks/tests -k unit
+            displayName: 'Run Unit Tests'
+            workingDirectory:  '$(self_repo_dir)'          
+
+              
+
```
Comment (motylp) [inline datastacks/datastacks-pipeline.yml:62]:
```suggestion
            workingDirectory: '$(self_repo_dir)'

```

File: datastacks/datastacks-pipeline.yml (hunk 1 above)
Comment (satenderrathee) [reply datastacks/datastacks-pipeline.yml]: removed

File: datastacks/datastacks-pipeline.yml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+              pip install pylint pylint-exit pytest-cov poetry
+            displayName: Install Pipeline Tools
+          - bash: |
+              poetry install 
+            displayName: Install Dependencies
+            workingDirectory: $(self_repo_dir)
+          - bash: |
+              poetry run pytest -rf -v datastacks/tests -k unit
+            displayName: 'Run Unit Tests'
+            workingDirectory:  '$(self_repo_dir)'
```
Comment (motylp) [inline datastacks/datastacks-pipeline.yml:55]:
```suggestion
            workingDirectory: '$(self_repo_dir)'
```

**Review by motylp — APPROVED** [review]

**Pull request merged**